src/NuGet.Clients/NuGet.SolutionRestoreManager/BrokeredServicesUtility.cs

```diff
@@ -11,11 +11,19 @@ namespace NuGet.SolutionRestoreManager
     internal static class BrokeredServicesUtility
     {
         // It is very important that these names and versions are kept the same as the ones in NuGet.VisualStudio.Internal.Contracts.NuGetServices.
-        internal const string NuGetSolutionServiceName = "NuGetSolutionService";
-        internal const string NuGetSolutionServiceVersion = "1.0.0";
+        internal const string SolutionServiceName = "Microsoft.VisualStudio.NuGet.SolutionService";
+        internal const string SolutionServiceVersion = "1.0.0";
 
-        internal static readonly ServiceRpcDescriptor NuGetSolutionService = new ServiceJsonRpcDescriptor(
-            new ServiceMoniker(NuGetSolutionServiceName, new Version(NuGetSolutionServiceVersion)),
+        internal const string DeprecatedSolutionServiceName = "NuGetSolutionService";
```

Comment:
We might be able to remove this in the future, but I am not up to date on the guidance here. 

We're still in an internal preview stage, so I think making breaking changes eventually is easier at this point than it would be once everything ships.

Reply:
Agree, will check to see if there is telemetry, if possible I will remove it earlier but for now we should support it.